src/Elastic.Documentation.Site/Assets/web-components/SearchOrAskAi/Search/SearchResults/SearchResultsList.tsx

```diff
@@ -34,16 +26,21 @@ export const SearchResultsList = ({
     buttonRef,
     itemRefs,
 }: SearchResultsListProps) => {
+    if (isLoading) {
+        return null
+    }
     const { euiTheme } = useEuiTheme()
     const selectedIndex = useSelectedIndex()
     const { setSelectedIndex, clearSelection } = useSearchActions()
     const scrollContainerRef = useRef<HTMLDivElement>(null)
 
     const scrollbarStyle = css`
         max-height: 400px;
-        padding-block: ${euiTheme.size.base};
-        margin-right: ${euiTheme.size.s};
+        padding-block-start: ${euiTheme.size.base};
+        padding-block-end: ${euiTheme.size.base};
+        margin-inline-end: ${euiTheme.size.s};
         ${useEuiOverflowScroll('y', true)}
```

Comment:
You can use `${useEuiOverflowScroll('y', false)}` or `${useEuiOverflowScroll('y')}` to remove the mask.